extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -196,7 +240,36 @@ class MongoQueueProcessor {
         entry.setLocation(editLocations);
     }
 
-    _updateReplicationInfo(entry) {
+    /**
+     * Update acl info on ingested object MD to match Zenko defined bucket acl.
+     * @param {ObjectQueueEntry} entry - object queue entry object
+     * @return {undefined}
+     */
+    _updateAcl(entry) {
+        // TODO: Rely on ObjectMD to set this
```

Comment:
Thinking out loud here: seems like it might be quicker to import instantiate `ObjectMD` and then call `getAcl`. That way we wouldn't have to revisit this.

Reply:
great idea, i've went ahead and made this change